pkg/controllers/repositoryserver/repository.go

```diff
@@ -52,8 +53,21 @@ func (h *RepoServerHandler) connectToKopiaRepository() error {
 	)
 }
 
-func (h *RepoServerHandler) getRepositoryCacheSettings() (contentCacheMB, metadataCacheMB int, err error) {
+func (h *RepoServerHandler) getRepositoryCommonArguments() (configFilePath, logDirectory string) {
+	configFilePath = command.DefaultConfigFilePath
+	logDirectory = command.DefaultLogDirectory
+	if h.RepositoryServer.Spec.Repository.ConfigFilePath != "" {
+		configFilePath = h.RepositoryServer.Spec.Repository.ConfigFilePath
+	}
+	if h.RepositoryServer.Spec.Repository.LogDirectory != "" {
+		logDirectory = h.RepositoryServer.Spec.Repository.LogDirectory
+	}
+	return
```

Comment:
```suggestion
	return configFilePath, logDirectory
```
We should avoid naked returns since it forces the one changing code later to read the code very carefully and increases chances of error.